src/Orleans/Streams/SimpleMessageStream/SimpleMessageStreamProducer.cs

```diff
@@ -32,12 +33,14 @@ internal class SimpleMessageStreamProducer<T> : IInternalAsyncBatchObserver<T>
         private readonly Logger                         logger;
         [NonSerialized]
         private readonly AsyncLock                      initLock;
-
+        [NonSerialized]
+        private readonly ILoggerFactory loggerFactory;
         internal bool IsRewindable { get; private set; }
 
         internal SimpleMessageStreamProducer(StreamImpl<T> stream, string streamProviderName,
             IStreamProviderRuntime providerUtilities, bool fireAndForgetDelivery, bool optimizeForImmutableData,
-            IStreamPubSub pubSub, bool isRewindable, SerializationManager serializationManager)
+            IStreamPubSub pubSub, bool isRewindable, SerializationManager serializationManager,
+            ILoggerFactory loggerFac)
```

Comment:
Please don't abbreviate in that way, just use `loggerFactory` as param name. There's many occurrences of this